components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -378,6 +382,21 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                     po::value<uint64_t>(&m_max_num_results)->value_name("MAX_NUM_RESULTS")->
                         default_value(m_max_num_results),
                     "The maximum number of results to output"
+            )(
+                    "reducer-host",
```

Comment:
Not all of these are "Results Cache Output Options" so maybe we rename the options to "Output Options"?